recipe/pixi.toml

```diff
@@ -0,0 +1,18 @@
+[project]
+name = "repodata-patches"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["Wolf Vollprecht <w.vollprecht@gmail.com>"]
+channels = ["conda-forge", "nodefaults"]
+platforms = ["osx-arm64", "linux-64", "win-64"]
+
+[tasks]
+show-diff = "python ./show_diff.py"
+
+[dependencies]
+annotated-types = "*"
+conda-build = "*"
+license-expression = "*"
+pre-commit = "*"
```

Comment:
`pre-commit` isn't going to fire unless inside a `shell`... having a `task.lint` that runs all the linters against all the files would likely be helpful.